core/templates/components/on-screen-keyboard/on-screen-keyboard.component.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the on-screen keyboard used for math
+ * interactions.
+ */
+
+require('domain/utilities/url-interpolation.service.ts');
+require('services/contextual/device-info.service.ts');
+require('services/guppy-initialization.service.ts');
+
+angular.module('oppia').component('onScreenKeyboard', {
+  bindings: {
+    // These will be used after a customization arg has been added to the
+    // math interactions.
+    customizable: '=',
```

Comment:
I'm not actually sure about this yet. Since I haven't integrated the customization args yet so I don't know exactly how this would function. But, I will change this when I implement the customization args.